adapter/config/default_config.go

```diff
@@ -175,6 +175,15 @@ var defaultConfig = &Config{
 				EnableOutboundCertificateHeader: false,
 			},
 		},
+		Cors: cors{
+			Enabled:                       true,
```

Comment:
are we enabling cors default for all apis? I rem like we disable cors by default. (enbling cors for unnecesary apis will add a computational overhead)

Reply:
this applies when v1 to v2 migration of apipolicy.